build/main.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+
+export BRANCH=$1
+export PACKAGE_JOB=$2
+export PACKAGE_UPLOAD_DIRECTORY=$3
+export PACKAGE_BUILD=$4
+
+export JOB_TO_UPLOAD=$PACKAGE_JOB
+export FLAG_FILE_URL="http://buildcache.cfengine.com/packages/$PACKAGE_JOB/$PACKAGE_UPLOAD_DIRECTORY/PACKAGES_HUB_x86_64_linux_ubuntu_16/core-commitID"
+export NO_OUTPUT_DIR=1
+
+env
+set -x
+
+# take ownersip of all files
+sudo chown -R jenkins:jenkins .
+
+export WRKDIR=`pwd`
+
+cd $WRKDIR/documentation-generator
+
+
+### download CFEngine ###
+
+# c https://github.com/cfengine/misc/blob/master/vagrant_quickstart/build.sh
```

Comment:
?

Reply:
where this function was copied from?

Or a reminder to share improvements with that script, too:
https://github.com/cfengine/misc/pull/146